### PR TITLE
Bookstore: use Redux in React components.

### PR DESCRIPTION
**Pull request by abdullah-asghar-khan**
Added three default books to the initial state.
Imported the Redux Provider and your store in the main component (App).
Set the data inputs in the Book component local React state.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -4,7 +4,24 @@ const BOOK_REMOVED = 'BOOK_REMOVED';
...
+    author: 'Oshani Chokshi',
+    id: '2',
+  },
+  {
+    title: 'The House of Broken Angels',
+    author: 'Luis Alberto',
+    id: '3',
+  },
+
+];
```
Comment (BenMukebo) [inline src/redux/books/books.js:24]: - [x] - [ ]  Your app runs with errors. I'm not able to see the BOOKS page.  After running the `npm install` then `npm start`.  Actually, I'm getting this error message 👇🏻 Please ensure that your app works as expected and runs without errors. 😉 👍🏻 

- Browser: 
15SCR

- Console:

15scc

- [x] when I close the error message page, I noticed that the book is not deleted correctly, and sometimes clicking a delete book button deletes all books in the list. please fix that too 👍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Abdullah2213565,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Descriptive PR ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ OPTIONAL] You can import `applyMiddleware` - to use some handy Redux middleware, like a redux logger.
  * it will help you a lot in the development process. To install it, the student needs to run:

    ```
    npm i --save redux-logger
    ```


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdullah-asghar-khan** [comment]
Hi @BenMukebo .
In my browser I am not having any error like that, it´s successfully compiled and it´s working.
Screenshot (1)
Screenshot (2)

**Comment by BenMukebo** [comment]
#### @Abdullah2213565 , Please request a second review, the next code reviewer will check and if everything works fine as shown in the screenshot above 🙃 , he will approve your project. 👍🏻

**Comment by abdullah-asghar-khan** [comment]
Thanks for your reply @BenMukebo .

**Comment by BenMukebo** [comment]
Great job 👍🏻 I can run your app without errors


- [x] but the remove button still does not work correctly, you need to fix this

**Comment by abdullah-asghar-khan** [comment]
For me the remove button is working correctly 😅

**Comment by BenMukebo** [comment]
😄 Alright, let's see what the next code reviewer has to say!!
Maybe the problem comes from my computer 🤔

**Comment by julie-ify** [comment]
Everything seems fine on my end as well. I am able to remove and add books

**Review by julie-ify — APPROVED** [review]
Hi @Abdullah2213565,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenMukebo** [comment]
###  Thank you @julie-ify 🙏🏻 🙏🏻

**Pull request merged**